python/ray/serve/experimental/gradio_visualize_graph.py

```diff
@@ -127,11 +198,17 @@ async def _get_result(self, node_uuid: str):
         This function should only be called after a request has been sent through
         self._send_request() separately.
         """
-        result = await self.cache[node_uuid]
-        self.resolved_nodes += 1
-        if self.resolved_nodes == len(self.node_to_block):
-            self.finished_last_inference = True
-        return result
+        try:
+            result = await self.cache[node_uuid]
+            self.resolved_nodes += 1
+            if self.resolved_nodes == len(self.node_to_block):
+                self.finished_last_inference = True
+            return postprocessing(result)
+        except BaseException:
```

Comment:
What is a `BaseException`? What causes it?

Reply:
All exceptions derive from `BaseException`.

Reply:
Ah, I see. Can we use `Exception` instead? All exceptions, [including system-exiting exceptions](https://stackoverflow.com/a/31609619) like `KeyboardInterrupt` (see thread in link for more info), inherit from `BaseException`.

Reply:
Changed in the original PR as it is unrelated to this PR: https://github.com/ray-project/ray/pull/27897